runtime/j9vm/javanextvmi.cpp

```diff
@@ -410,6 +407,12 @@ JVM_VirtualThreadMountEnd(JNIEnv *env, jobject thread, jboolean firstMount)
 release2:
 	f_monitorExit(vm->liveVirtualThreadListMutex);
 release1:
+	if (NULL != vm->liveVirtualThreadList) {
```

Comment:
This can never be NULL at this point, I believe.

Reply:
It won't always be NULL if there are errors presumably, so you need some other logic (like a boolean variable) to decide if the hooks should be fired.

Reply:
Added a boolean to trigger the J9Hooks only if no errors were encountered.